RATapi/utils/convert.py

```diff
@@ -549,9 +549,10 @@ def convert_parameters(
     # scipy.io.savemat doesn't do cells properly:
     # https://github.com/scipy/scipy/issues/3756
     # rather than fiddling we just use matlab
-    eng = wrappers.start_matlab().result()
+    eng = wrappers.start_matlab()
```

Comment:
We have to refactor this in the future as this is loading up another instance of MATLAB on every call instead of taking advantage of the one loaded in the `MatlabWrapper` class. A quick workaround would be to do the following instead

```
    loader = wrappers.MatlabWrapper.loader
    if loader is None:
        raise ImportError("matlabengine is not installed.")
    eng = loader.result()
    eng.workspace["problem"] = r1
    eng.save(str(filename), "problem", nargout=0)
    return None
```

Reply:
Shall we do this now and then see where we are later?